core/src/main/java/org/elasticsearch/rest/action/cat/RestTable.java

```diff
@@ -109,6 +114,31 @@ public static RestResponse buildTextPlainResponse(Table table, RestChannel chann
         return new BytesRestResponse(RestStatus.OK, BytesRestResponse.TEXT_CONTENT_TYPE, bytesOut.bytes());
     }
 
+    static List<Integer> getRowOrder(Table table, RestRequest request) {
+        String[] columnOrdering = request.paramAsStringArray("s", null);
+
+        List<Integer> rowOrder = new ArrayList<>();
+        for(int i = 0; i < table.getRows().size(); i++) {
```

Comment:
nit: we usually add a space here. We don't have anything that enforces this style but we usually do.